src/sdk/AutoDetectSourceLanguageConfig.ts

```diff
@@ -88,4 +88,21 @@ export class AutoDetectSourceLanguageConfig {
         return this.privProperties;
     }
 
+    /**
+     * @member AutoDetectSourceLanguageConfig.prototype.continuousMode
+     * @function
+     * @public
+     * @param {boolean} true if continuous mode LID desired.
+     * @summary Sets LID operation to continuous mode
+     */
+    public set continuousMode(isContinuous: boolean) {
+        if (isContinuous) {
+            this.privProperties.setProperty(PropertyId.SpeechServiceConnection_RecognitionEndpointVersion, "2");
+            this.privProperties.setProperty(PropertyId.SpeechServiceConnection_LanguageIdMode, "DetectContinuous");
+        } else {
+            this.privProperties.setProperty(PropertyId.SpeechServiceConnection_RecognitionEndpointVersion, "1");
+            this.privProperties.setProperty(PropertyId.SpeechServiceConnection_LanguageIdMode, "DetectAtAudioStart");
+        }
```

Comment:
What about setting the priority? "Latency" (the default) and "Accuracy"?

Reply:
Also, don't you want to keep JS property names and meanings the same as in rest of Carbon? Looks like here you defined a new property name that only deals with at-start/continuous.

Reply:
Priority has now been added. Since this is only continuous LID, only "Latency"/"PrioritizeLatency" or a blank string are allowed. When I add the full matrix of LID possibilities, "Accuracy" will become a possible value.